pkg/types/openstack/machinepool.go

```diff
@@ -6,6 +6,9 @@ type MachinePool struct {
 	// FlavorName defines the OpenStack Nova flavor.
 	// eg. m1.large
 	FlavorName string `json:"type"`
+
+	// RootVolume defines the root volume for instances in the machine pool.
+	RootVolume *RootVolume `json:"rootVolume,omitempty"`
```

Comment:
needs the information about `if optional` and what is the default when nil

Reply:
this is unset when nil, can I just leave it at that

Reply:
> https://github.com/openshift/installer/pull/2394/files#diff-e32975a988b75f5459f56a7c4e5affe7R19
`The instances use ephemeral disks if not set.` should be enough.

Reply:
Done